src/case_data.c

```diff
@@ -47,6 +47,130 @@ const u16 gCaseGameoverSections[] = {
     0xF7
 };
 
+// BEGIN RODATA
```

Comment:
the rodata is the two arrays at the start of the file

Reply:
welp, looks like i left those in from when i was trying to figure out what should go where